generator/recipe.yaml

```diff
@@ -1,9 +1,10 @@
 name: {{.Name}}
 version: {{.Version}}
 source:
-{{- range $key, $value := .Source }}
-  type: {{$key}}
-  config: {{$value | indent 4}}
+{{- with .Source }}
+  type: {{.Name}}
```

Comment:
oh here is as well, we can change `type` to `name`. 🙏

Reply:
Done, apologies for missing it earlier.